src/config/config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "accuracy_cutoff": 50,
+  "response_time_limit": 5000
+}
```

Comment:
This is the default right? I think @mcmcgrath13 said that we want to keep the old values as default, so 80 and 1250.

Reply:
Oh oops okay thanks!